pkgs/os-specific/linux/setools/default.nix

```diff
@@ -8,16 +8,16 @@ with python3.pkgs;
 
 buildPythonApplication rec {
   pname = "setools";
-  version = "4.2.0";
+  version = "4.2.2";
 
   src = fetchFromGitHub {
     owner = "SELinuxProject";
     repo = pname;
     rev = version;
-    sha256 = "1bjwcvr6rjx79cdcvaxn68bdrnl4f2a8gnnqsngdxhkhwpddksjy";
+    sha256 = "18kklv26dwm2fdjjzfflvxsq83b2svnwf4g18xq7wsfsri121a90";
   };
 
-  nativeBuildInputs = [ bison flex swig ];
+  nativeBuildInputs = [ cython ];
   buildInputs = [ libsepol ];
   propagatedBuildInputs = [ enum34 libselinux networkx ]
```

Comment:
```suggestion
  propagatedBuildInputs = [ enum34 libselinux networkx setuptools ]
```
```
[nix-shell:~/.cache/nix-review/pr-68728]$ ./results/setools/bin/seinfo --help
Traceback (most recent call last):
  File "/nix/store/l2y3j9bqzd632p1asafxp6cs48ipd2wb-setools-4.2.2/bin/.seinfo-wrapped", line 22, in <module>
    import setools
  File "/nix/store/l2y3j9bqzd632p1asafxp6cs48ipd2wb-setools-4.2.2/lib/python3.7/site-packages/setools/__init__.py", line 76, in <module>
    from .infoflow import InfoFlowAnalysis
  File "/nix/store/l2y3j9bqzd632p1asafxp6cs48ipd2wb-setools-4.2.2/lib/python3.7/site-packages/setools/infoflow.py", line 23, in <module>
    import networkx as nx
  File "/nix/store/m0nlk8hwb1fd1dn1aaiwv09wqjmcgyxq-python3.7-networkx-2.2/lib/python3.7/site-packages/networkx/__init__.py", line 128, in <module>
    import networkx.drawing
  File "/nix/store/m0nlk8hwb1fd1dn1aaiwv09wqjmcgyxq-python3.7-networkx-2.2/lib/python3.7/site-packages/networkx/drawing/__init__.py", line 6, in <module>
    from . import nx_pydot
  File "/nix/store/m0nlk8hwb1fd1dn1aaiwv09wqjmcgyxq-python3.7-networkx-2.2/lib/python3.7/site-packages/networkx/drawing/nx_pydot.py", line 27, in <module>
    from pkg_resources import parse_version
ModuleNotFoundError: No module named 'pkg_resources'
```

Reply:
Its linked in the related issue, this is within networkx.

Reply:
This one: https://github.com/NixOS/nixpkgs/pull/68727
In reality this probably goes deeper, as the networkx packaged does not actually provide either pygraphviz or pydot, so likely the networkx package needs to be improved a little bit.